lib/components/src/navigation/menu_link.js

```diff
@@ -0,0 +1,23 @@
+import glamorous from 'glamorous';
+import RoutedLink from './routed_link';
+
+export default glamorous(RoutedLink, { rootEl: 'a' })(
+  {
+    display: 'block',
+    color: '#828282',
```

Comment:
Should we worry about theming yet in this PR?

I'd like to indeed get theming support, and use https://glamorous.rocks/advanced#theming for this.